apps/et/preview/aso/et-postgres.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: dbforpostgresql.azure.com/v1api20210601
+kind: FlexibleServer
+metadata:
+  name: ${NAMESPACE}-flexibleserver-postgres-${ENVIRONMENT}
+  namespace: ${NAMESPACE}
+  annotations:
+    serviceoperator.azure.com/reconcile-policy: detach-on-delete
+spec:
+  owner:
+    name: ${NAMESPACE}-aso-${ENVIRONMENT}-rg  #created ahead and added to flux
+  location: uksouth
+  version: "11"
```

Comment:
would it be better to configure this to v14?

Reply:
```suggestion
  version: "15"
```